evap/results/tests/test_views.py

```diff
@@ -264,6 +265,38 @@ def make_course_with_evaluations(unique_suffix):
         caches["sessions"].clear()
         caches["results"].clear()
 
+    def test_evaluation_weight_sums(self):
+        """
+        ensures that the sum of evaluation weights add up to 100%,
+        even if some evaluations are not shown to the user
```

Comment:
In this test, the `student` user can see all the evaluations, right? That is a mismatch to this comment. If I remember correctly, the wrong display occurred when users could not see all evaluations(?), so it might make sense to make some of these evaluations not have results for the user. I'd suggest having a few in the "new" state.